alpha/declcfg/declcfg.go

```diff
@@ -16,16 +16,18 @@ import (
 )
 
 const (
-	SchemaPackage = "olm.package"
-	SchemaChannel = "olm.channel"
-	SchemaBundle  = "olm.bundle"
+	SchemaPackage     = "olm.package"
+	SchemaChannel     = "olm.channel"
+	SchemaBundle      = "olm.bundle"
+	SchemaDeprecation = "olm.catalog.deprecation"
```

Comment:
Did we ever debate what the name of this should be? The most intuitive name that comes to mind for me is is `olm.deprecations`, but I can't remember if we had `"catalog"` in there for a reason.